test/e2e/validation_test.go

```diff
@@ -21,30 +20,34 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	kubeTypes "k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/client-go/kubernetes"
 )
 
 // waitForTrackerConfigMap to be created waits for the Windows tracker configmap to be created with appropriate values
-func waitForTrackerConfigMap(kubeclient kubernetes.Interface, namespace string, expectedNodesToBeTracked int,
-	retryInterval, timeout time.Duration) error {
-	err := wait.Poll(retryInterval, timeout, func() (done bool, err error) {
-		trackerConfigMap, err := kubeclient.CoreV1().ConfigMaps(namespace).Get(tracker.StoreName, metav1.GetOptions{})
+func (t *testContext) waitForTrackerConfigMap() error {
+	namespace, err := t.osdkTestCtx.GetNamespace()
+	if err != nil {
+		return err
+	}
```

Comment:
Why can't the `namespace` be part of the `TestContext`? Otherwise the code is going to littered with this.

Reply:
I'm +1 on moving namespace to `TestContext`. Theres a difference and reason to move it to the parent context, as it could be set to a varaibel and we would not have to access the field through a getter.